persistence/selecting.md

```diff
@@ -58,9 +60,11 @@ The non-durable persister is not mentioned in the decision chart, as it is appro
 
 ### Azure
 
-There are several options available when endpoints are hosted in Windows Azure. The most commonly used persisters are Azure Storage and Azure SQL, the fully managed SQL Server solution in Azure.
+There are several options available when endpoints are hosted in Microsoft Azure. The most commonly used persisters are Azure Storage, Azure CosmosDB and Azure SQL, the fully managed SQL Server solution in Azure.
 
-One factor in the decision is whether the system is fully platform-as-a-service-enabled and whether it is designed to run fully on Azure (in which case, the Azure Table persistence may be appropriate). Alternatively, some organizations are more comfortable managing SQL Server and may choose an infrastructure-as-a-service solution (using the SQL persister with Azure SQL).
+One factor in the decision is whether the system is fully platform-as-a-service-enabled and whether it is designed to run fully on Azure in which case, the Azure Storage Table or Azure CosmosDB persistence may be appropriate. The choice between Azure Storage Table and Azure CosmosDB boils down to a difference in cost vs capabilities.
+
+Alternatively, some organizations are more comfortable managing SQL Server and may choose an infrastructure-as-a-service solution (using the SQL persister with Azure SQL).
```

Comment:
I know this sentence was already there but frankly, I find it confusing. Above we say

> Azure SQL, the fully managed SQL Server solution in Azure

and then here we say

> and may choose an infrastructure-as-a-service solution (using the SQL persister with Azure SQL).

Reply:
perhaps we can even add 'Sql Server hosted on VM'?

Reply:
I think it would be more precise and less confusing

Reply:
done